tests/posix/wcwidth.c

```diff
@@ -1,10 +1,17 @@
 #include <assert.h>
 #include <locale.h>
+#include <stdint.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <wchar.h>
 
+#if UINTPTR_MAX == UINT64_MAX
+#define WCHAR_SPEC ""
+#else
+#define WCHAR_SPEC "l"
+#endif
+
```

Comment:
Why do we need this? A `wchar_t` is 32-bit either way

Reply:
GCC was complaining about some `-Wformat` warnings and that fixed it.